### ISSUE TITLE
Improve REUSE recommendations

### ISSUE THREAD

**Issue by andreashaerter** [foundata/guidelines#7]
Our current approach to introduce REUSE is

1. `.reuse/dep5` with `Files: *` stanza
2. Use header comment in files with a high profitability to be used stand-alone, e.g.  script files
3. The header *must* state the same license as the default or an explicit entry in `.reuse/dep5`

The main reasoning for this was (after discussions):

1. Having a compliant default is a good machine readable staring point and far better than simple `LICENSE` files + README.md entry, especially as preparation for multiple licenses
2. the `reuse lint` tool complains with a deprecation warning even if the license  in `.reuse/dep5` and the inline header is *exactly* the same.

We like the default as it prevents the need to put license information for very few, *insignificant* files and tooling directories while still making it technically easy to determine the license plus putting license headers in *nearly* every file in the future.

However, our `.reuse/dep5` with `Files: *` stanza-default seems to be strongly discouraged (see https://github.com/fsfe/reuse-docs/issues/117, especially [one comment](https://github.com/fsfe/reuse-docs/issues/117#issuecomment-1306963966)). We therefore should discuss how to proceed and how to adapt [`licensing-how-to-apply.md`](https://github.com/foundata/guidelines/blob/master/licensing-how-to-apply.md).

Additionally, [the Gem used by GitHub](https://github.com/licensee/licensee/issues/490) and [others](https://forum.openmod.org/t/reuse-incompatible-to-auto-detection-of-license-s/3590) is still not able to detect the license of our repositories automatically because of this. Symlinks *might* be the solution (but this is a good example why `Files: *` is so pleasant... it is clumsy to have a `LICENSE` file because of licensee-Gem issues even if a `LICENSE.license` is [not needed](https://github.com/fsfe/reuse-tool/issues/881) as [REUSE ignores a `COPYING` or `LICENSE` file in the root](https://reuse.software/faq/#tradition)).

[Licensee even does not support symlinks](https://github.com/licensee/licensee/pull/42) (while `reuse lint` does), so one could only put a `LICENSE` file in the project root and create a Symlink to it in the `LICENSES` directory to prevent duplication.

**Comment by andreashaerter** [comment]
It is in fact no problem to define stanzas for non-existing files in the `dep5` file. Providing a sane template is probably a good way to solve the issue.

**Comment by andreashaerter** [comment]
Closing this now for the time being after new discussions and improvements. Summary

- The stanza using a wildcard as default makes sense, the deprecation warning seems to be a bug (cf. [issue 885](https://github.com/fsfe/reuse-tool/issues/885))
- We changed our current [recommendation](https://github.com/foundata/guidelines/blob/master/licensing-how-to-apply.md#license-comment-headers) and will add license comment headers in most files (cf. fab6bf8c44bbe02221a28e788c4e40d4f71d8aff)
